python/python_fastapi_server/routers/edr.py

```diff
@@ -412,7 +447,7 @@ async def get_collectioninfo_for_id(
     else:
         data_queries = DataQueries(position=EDRQuery(link=position_link))
 
-    parameter_names = get_params_for_collection(edr_collection=edr_collection)
+    parameter_names = get_params_for_collection(edr_collection=edr_collection, wmslayers=wmslayers)
```

Comment:
Unclear name for this variable. It's a dict that holds more than names.

Reply:
It is just the list of wmslayers from the getcapabilities with names and dimensions. Not sure what a better name would be.